Makefile

```diff
@@ -202,7 +202,7 @@ bpf/lint-fix:
 	$(MAKE) -C bpf lint-fix
 
 test/profiler: $(GO_SRC) $(LIBBPF_HEADERS) $(LIBBPF_OBJ) bpf
-	sudo $(GO_ENV) $(CGO_ENV) $(GO) test $(SANITIZERS) -v ./pkg/profiler/...
+	sudo $(GO_ENV) $(CGO_ENV) $(GO) test $(SANITIZERS) -v ./pkg/profiler/... -count=1
```

Comment:
Avoid the aggressive caching by go's testing framework. We want to ensure we run these tests all the time, even if no source-level changes are made.